src/consent.ts

```diff
@@ -8,10 +8,14 @@ import { TelemetryOptions } from './types'
 import { consentVersion } from './meta'
 
 export async function ensureUserconsent (options: TelemetryOptions): Promise<boolean> {
-  if (options.consent >= consentVersion || stdEnv.minimal || process.env.CODESANDBOX_SSE || isDocker()) {
+  if (options.consent >= consentVersion) {
     return true
   }
 
+  if (stdEnv.minimal || process.env.CODESANDBOX_SSE || isDocker()) {
```

Comment:
I think we could move CODESANDBOX to line 11 since it is only in development for it

Reply:
Actually we may enable it by .nuxtrc in template.. I don't want to open yet another flow of auto opt-in even though CSB is mainly for development (like im sure it is also possible to use it for private sandboxes)

Reply:
Makes perfect sense :)